runelite-client/src/main/java/net/runelite/client/plugins/ammo/AmmoPlugin.java

```diff
@@ -90,16 +95,26 @@ public void onItemContainerChanged(ItemContainerChanged event)
 			return;
 		}
 
-		checkInventory(event.getItemContainer().getItems());
+		checkInventory(event.getItemContainer());
+	}
+
+	@Subscribe
+	public void onGameTick(GameTick event)
+	{
+		checkQuiver();
 	}
 
-	private void checkInventory(final Item[] items)
+	private void checkInventory(ItemContainer equipment)
 	{
+		final Item cape = equipment.getItem(EquipmentInventorySlot.CAPE.getSlotIdx());
+		this.isWearingQuiver = cape != null && itemManager.getItemComposition(cape.getId()).getName().startsWith("Dizana");

```

Comment:
I'd rather use ItemVariationMapping to check for the cape being the quiver or quiver max cape explicitly here.